boards/shields/adafruit_2_8_tft_touch_v2/dts_fixup.h

```diff
@@ -0,0 +1 @@
+#define DT_DISPLAY_SHIELD_DEV_NAME DT_INST_0_ILITEK_ILI9340_LABEL
```

Comment:
I suggest to use CONFIG_LVGL_DISPLAY_DEV_NAME as it is already there.

Reply:
That is not an option `CONFIG_LVGL_DISPLAY_DEV_NAME` can only be set if LVGL is used else this config is not available.